test/OpenTelemetry.Extensions.Tests/Trace/BaggageActivityProcessorTests.cs

```diff
@@ -0,0 +1,115 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Text.RegularExpressions;
+using OpenTelemetry.Tests;
+using OpenTelemetry.Trace;
+using Xunit;
+
+namespace OpenTelemetry.Extensions.Tests.Trace;
+
+public class BaggageActivityProcessorTests
+{
+    [Fact]
+    public void BaggageActivityProcessor_CanAddAlloAllBaggageKeysPredicate()
+    {
+        var activityProcessor = new TestActivityProcessor();
```

Comment:
not sure what is the need of `TestActivityProcessor` in this test? Most likely a leftover from other tests?

Reply:
Removed in [19fda8c](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1816/commits/19fda8cc0eea9d6d9af678672b8199455d9a1c9c)